apps/web/src/components/Resource/Contents/LinkView.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames'
+import Link from 'next/link'
+import { ResourceContent } from '@app/web/server/resources/getResource'
+import styles from './LinkView.module.css'
+
+const LinkView = ({
+  content: { title, caption, showPreview, url, linkTitle, linkDescription },
+}: {
+  content: Pick<
+    ResourceContent,
+    | 'title'
+    | 'caption'
+    | 'showPreview'
+    | 'url'
+    | 'linkTitle'
+    | 'linkDescription'
+  >
+}) => (
+  <div data-testid="content-link-title">
+    <h6 className="fr-mb-2w">{title}</h6>
+    {showPreview && (
```

Comment:
Avec les legacy on ne peut pas faire confiance à showPreview, je pense qu’il faut ajouter un test s
`ShowPreview and !!title` par ex et prévoir une description ou image null egalement

Reply:
yes, à voir quand on aura des exemples complets et des legacy, je laisse comme ca pour l'instant